Libplanet.Tests/KeyStore/Ciphers/Aes128CtrTest.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Immutable;
+using System.Text.Json;
+using Libplanet.KeyStore;
+using Libplanet.KeyStore.Ciphers;
+using Libplanet.KeyStore.Kdfs;
+using Org.BouncyCastle.Crypto.Digests;
+using Xunit;
+
+namespace Libplanet.Tests.KeyStore.Ciphers
+{
+    public class Aes128CtrTest : CipherTest<Aes128Ctr>
+    {
+        private Aes128Ctr _cipher;
+
+        public Aes128CtrTest()
+        {
+            var random = new Random();
+            var buffer = new byte[16];
+            random.NextBytes(buffer);
+            _cipher = new Aes128Ctr(buffer.ToImmutableArray());
+        }
+
+        public override Aes128Ctr Cipher => _cipher;
```

Comment:
This can be converted to `public override Aes128Ctr Cipher { get; }`

Reply:
Fixed: a83f6a606a503e40e5a78db9f00d43b47d9168d6.